participants/Music-Wiki.md

```diff
@@ -0,0 +1,58 @@
+# Music Wiki
+
+## (0) Project Name and Team Slack Handles
+
+Project Name:Music Wiki  
+Slack:Zyan Wlay
+
+## (1) Brief Project Description
+
+*The Music Wiki is a free music retrieval service website, using filcoin network to store music data,and other rapid storage to provider a fast retrieval service.*
+
+## (2) Link to Project UI
+
+http://113.31.119.110
+(Submission ID: 674625967)
+
+## (3) What does your application/UI do?
+
+*The website can receive music uploaded by people.Everyone can improve our music catalogue,everyone can find search music through name.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*redis and mysql*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
```

Comment:
Do you have the rights to this data or is all the music uploaded copyright free?

Reply:
There is no complete data in our application, and the data uploaded to the IPFS network will be reviewed by another copyritht checking model, later.